client/src/components/professional/index.js

```diff
@@ -15,93 +14,152 @@ import './style.css'
-        <div className="professionalUrl">
-          <div>
-            <LanguageIcon style={orgPage.urlIcon} />
           </div>
-          <div>
-            <p>www.linkedin.com/in/[vanityname]</p>
+          <div className="professionalDescription">
+            <Divider />
+            <p>
+              {description}
+            </p>
+            {hasVideo && (
+              <iframe 
+                className="professionalVideo" 
+                title="Professional Video" 
+                allow= "encrypted-media"
+                src={videoUrl}
+              />
+            )}
+            <Divider />
           </div>
-        </div>
-      </div>
-      <div className="professionalDescription">
-        <Divider />
-        <p>Description</p>
-        {hasVideo && (
-          <div>
-            <video /> {/* placeholder, has linting error here */}
+          <div className="professionalExperiences">
+            <List>
+              {createExperienceTable(experiences)}
+            </List>
           </div>
-        )}
-        <Divider />
-      </div>
-      <div className="professionalExperiences">
-        <List>
-          <ListItem leftIcon={<Domain />}>
-            <h4>[Organization name] - [Job Title]</h4>
-            <p>[start date] - [end date]</p>
-            <p>[Award(s)]</p>
-            <ListItem leftIcon={<Group />}>
-              <h4>[Program] - [Job Title]</h4>
-              <p>[Start Date] - [End Date]</p>
-              <p>[Award(s)]</p>
-            </ListItem>
-            <ListItem leftIcon={<Group />}>
-              <h4>[Program] - [Job Title]</h4>
-              <p>[Start Date] - [End Date]</p>
-              <p>[Award(s)]</p>
-            </ListItem>
-          </ListItem>
-          <ListItem leftIcon={<Domain />}>
-            <h4>[Organization name] - [Job Title]</h4>
-            <p>[start date] - [end date]</p>
-            <p>[Award(s)]</p>
-          </ListItem>
-          <ListItem leftIcon={<Domain />}>
-            <h4>[Organization name] - [Job Title]</h4>
-            <p>[start date] - [end date]</p>
-            <p>[Award(s)]</p>
-          </ListItem>
-          <ListItem leftIcon={<Domain />}>
-            <h4>[Organization name] - [Job Title]</h4>
-            <p>[start date] - [end date]</p>
-            <p>[Award(s)]</p>
-          </ListItem>
-        </List>
+        </div>
       </div>
-    </div>
-  </div>
-)
+    )
+  }
+}
+
+function createExperienceTable(experiences) {
+  return experiences.map( experience => {
+    const {
+      id,
+      startDate,
+      endDate,
+      title,
+      award,
+      organization,
+      program
+    } = experience
+
+    const name = organization ? organization.name : program.name
+    if (organization === undefined)
+      return createProgramTable(name, experience)
+
+    return (
+      <ListItem key={id} leftIcon={<Domain />}>
+        <h4>{name} - {title}</h4>
+        <p>{startDate} - {endDate}</p>
```

Comment:
We should parse these dates and convert them to readable dates like "Jan 2005 - May 2014" or "Jan 2005 - Present".